src/deserialization.hpp

```diff
@@ -200,4 +200,66 @@ Status deserializePredictRequest(
     }
     return status;
 }
+
+template <class TensorProtoDeserializator, class Sink>
+Status deserializePredictRequest(
+    const ::inference::ModelInferRequest& request,
+    const tensor_map_t& inputMap,
+    Sink& inputSink, bool isPipeline) {
+    OVMS_PROFILE_FUNCTION();
+    Status status;
+    for (const auto& pair : inputMap) {
+        try {
+            const auto& name = pair.first;
+            auto tensorInfo = pair.second;
+            auto requestInputItr = std::find_if(request.inputs().begin(), request.inputs().end(), [&name](const ::inference::ModelInferRequest::InferInputTensor& tensor) { return tensor.name() == name; });
+            if (requestInputItr == request.inputs().end()) {
+                SPDLOG_DEBUG("Failed to deserialize request. Validation of request failed");
+                return Status(StatusCode::INTERNAL_ERROR, "Failed to deserialize request");
```

Comment:
Needs more explainatory message, which input name is missing. We report it for TFS api